### PR TITLE
Update to tectonic-console-builder-v21

### PR DESCRIPTION
**Pull request by dtaylor113**

### CODE REVIEW

**Comment by spadgett** [comment]
/hold
until we push the new builder image

File: core-services/prevent-image-gc/daemonset.yaml [hunk 1]
```diff
@@ -160,6 +160,24 @@ spec:
...
+            set -euo pipefail
+
+            trap 'jobs -p | xargs -r kill || true; exit 0' TERM
+            while true; do
+              sleep 600 &
+              wait $!
+            done
+          image: registry.svc.ci.openshift.org/openshift/release:tectonic-console-builder-v21
+          imagePullPolicy: Always
+          name: consolebuilder20
```
Comment (spadgett) [inline core-services/prevent-image-gc/daemonset.yaml:177]:
```suggestion
          name: consolebuilder21
```

Comment (dtaylor113) [reply core-services/prevent-image-gc/daemonset.yaml]: ok, fixed

**Comment by spadgett** [comment]
/hold cancel

We have an image on quay now: https://quay.io/repository/coreos/tectonic-console-builder?tab=tags

**Comment by spadgett** [comment]
/lgtm cancel

> error parsing core-services/prevent-image-gc/daemonset.yaml: error converting YAML to JSON: yaml: line 174: did not find expected '-' indicator

File: core-services/prevent-image-gc/daemonset.yaml [hunk 2]
```diff
@@ -160,6 +160,24 @@ spec:
         resources:
           requests:
             cpu: 50m
+      - command:
+          - /bin/bash
```
Comment (spadgett) [inline core-services/prevent-image-gc/daemonset.yaml:164]: Wrong indentation?

**Comment by dtaylor113** [comment]
```
time="2020-08-19T22:12:28Z" level=error msg="Failed to set up dependencies" error="failed to ensure imagestreamtags in cluster build01: failed waiting for imagestreamtag openshift/release:tectonic-console-builder-v21 to appear: timed out waiting for the condition" org=openshift repo=release
ERROR: pj-rehearse: setup failure
```

**Comment by spadgett** [comment]
@dtaylor113 This still has the same YAML parsing error:

> oc apply -f core-services/prevent-image-gc/daemonset.yaml --dry-run: failed to apply\nW0820 16:55:07.876826     970 helpers.go:535] --dry-run is deprecated and can be replaced with --dry-run=client.\nerror: error parsing core-services/prevent-image-gc/daemonset.yaml: error converting YAML to JSON: yaml: line 174: did not find expected '-' indicator

**Pull request merged**